addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -108,6 +108,14 @@ MockServer.include({
             const { min_id, max_id, limit } = args;
             return this._mockRouteMailMessageInbox(min_id, max_id, limit);
         }
+        if (route === '/mail/message/add_reaction') {
+            const { message_id, content } = args;
+            return this._mockMailMessageAddMessageReaction(message_id, content, this.currentPartnerId);
+        }
+        if (route === '/mail/message/remove_reaction') {
+            const { message_id, content } = args;
+            return this._mockMailMessageRemoveMessageReaction(message_id, content, this.currentPartnerId);
+        }
```

Comment:
`currentPartnerId` is not passed as param in python so it should not be here iehter

Reply:
The problem actually is that you lack the mock of the controller.

You directly call the mock from the model (which needs to exist), but it should be called from within the mock of the controller to be created that takes care of this part.

Reply:
So should I use `currentPartnerId` inside my method to not pass it as a parameter or don't use it at all ?

Also, I don't understand your comment about the way the mock is called. Those mock methods are called because of the actions performed in the tests. And finally, what controller are you talking about ?

Reply:
@seb-odoo

Reply:
Everything in this file is made to match exactly what exists in python. All methods should have the same signature.
There is one method in discuss controller, and one method in message model. Both of these methods should appear here with the same code inside them.

Reply:
Done, I think ?